packages/app-mobile/components/NoteEditor/NoteEditor.tsx

```diff
@@ -273,9 +134,9 @@ function NoteEditor(props: Props, ref: any) {
 			};
 		} catch (e) {
 			window.ReactNativeWebView.postMessage("error:" + e.message + ": " + JSON.stringify(e))
-		} finally {
-			true;
 		}
+		
+		true;
```

Comment:
`try { ... } catch(e) { ... } finally { true; }` evaluates to `undefined`, not `true`. `react-native-webview` requires all injected JS evaluate to `true`.